code/style.css

```diff
@@ -1,7 +1,545 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  margin: 0;
+  font-family: 'Quicksand', sans-serif;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  background: #3E3E3E;
+  color: #ffffff;
+}
+
+
+/* ----------- TYPOGRAPHY ----------- */
+
+h1, 
+h2 {
+  font-family: 'Dancing Script', cursive;
+  margin: 0;
+  padding-bottom: 1em;
+  text-align: center;
+}
+
+h1 {
+  font-size: 3.5rem;
+}
+
+h2 {
+  font-size: 2em;
+}
+
+h3 {
+  font-family: 'Bebas Neue', cursive;
+  letter-spacing: 1.7px;
+}
+
+.small-text {
+  font-size: 0.825rem;
+  font-weight: 500;
+}
+
+button,
+.modal-btn {
+  font-family: 'Bebas Neue', cursive;
+  letter-spacing: 1.6px;
+  font-size: 1em;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.infotext p {
+  font-family: 'Bebas Neue', cursive;
```

Comment:
You have repeatedly used these codes for your h3, button, modal-btn, infortext p. 
 `font-family: "Bebas Neue"`
` line-height: 1.6em`

I would reccommed to create a cusomized class for text styling and add them in HTML tags where you want to have the style. For example:
```
   .bebas-font {
      font-family: 'Bebas Neue', cursive;
   letter-spacing: 1.6px; }
```
   By this way, we can advoid repeating the code.

Reply:
This is smart! I will definitely work more with customized classes ahead, thank you! 😃